apps/issuer/models.py

```diff
@@ -356,7 +356,9 @@ def ensure_owner(self):
         elif staff.exists():
             new_owner = staff.first()
         else:
-            # Nothing we can do
+            # If there is no other user, we (re-)assign the creator as owner.
+            # This is also the case for the initial creation
+            new_owner = IssuerStaff.objects.create(issuer=self, user=self.created_by, role=IssuerStaff.ROLE_OWNER)
```

Comment:
This line made it impossible to remove issuer-creator, only new members can be removed.

My suggestion is to revert this PR to previous commit so we can merge it then we can create a new ticket for `deleting user from admin panel` as this ticket was created for removing owner.

Reply:
@YouQam I fixed that (the problem was actually in line 313). And I disagree that we should revert it, since it's quite intertwined and I don't think it makes sense to revert the fixes from the last commit(s).